src/modules/core/components/NetworkStatus/NetworkStatus.tsx

```diff
@@ -0,0 +1,176 @@
+      if (latestRpcBlockError) {
+        setNetworkHealth('critical');
+      } else if (
+        latestSubgraphBlockError ||
+        !isReputationOracleAlive?.isReputationOracleAlive ||
+        !isColonyServerAlive?.isServerAlive ||
+        (latestRpcBlock &&
+          latestSubgraphBlock &&
+          latestRpcBlock.latestRpcBlock >
+            latestSubgraphBlock.latestSubgraphBlock)
+      ) {
+        setNetworkHealth('poor');
+      } else {
+        // If everything is okay, set health to healthy (to correct for the previous state)
+        setNetworkHealth('healthy');
+      }
+    }, networkCheckInterval);
+    return () => clearInterval(networkCheckTwo);
+  });
+
+  const [showLoading, setShowLoading] = useState(true);
+  // 1 second for the spinner to show
+  const loadingDuration = 1 * 1000;
+
+  useEffect(() => {
+    const loadingTimer = setTimeout(() => {
+      setShowLoading(false);
+    }, loadingDuration);
+    return () => clearTimeout(loadingTimer);
+  }, [loadingDuration]);
+  return (
+    <>
+      {networkHealth !== 'healthy' &&
+        (showLoading ? (
+          <div className={`${styles.elementWrapper}`}>
+            <MiniSpinnerLoader />
+          </div>
+        ) : (
+          <div>
+            <Popover
+              appearance={{ theme: 'grey' }}
+              showArrow={false}
+              placement="bottom-start"
+              content={() => (
+                <div className={styles.networkHealth}>
+                  <div className={styles.networkHealthHeading}>
+                    <span
+                      className={cx(
+                        styles[`networkHealthIcon-${networkHealth}`],
+                        styles.networkHealthIcon,
+                      )}
+                    >
+                      <Icon name="triangle-warning" />
+                    </span>
+                    <span>
+                      {formatMessage(MSG.networkHealthHeader, {
+                        networkHealth,
+                      })}
+                    </span>
+                  </div>
+                  <span className={styles.networkHealthDescription}>
+                    {formatMessage(MSG.networkHealthDescription, {
+                      networkHealth,
+                    })}
+                  </span>
+
+                  <ExternalLink
+                    text={{ id: 'text.learnMore' }}
+                    className={styles.link}
+                    href={NETWORK_HEALTH}
+                  />
+                </div>
+              )}
+              popperProps={{
+                modifiers: [
+                  {
+                    name: 'offset',
+                    options: { offset: [0, 9] },
+                  },
+                ],
+              }}
+            >
+              <div
+                className={`${styles.elementWrapper}
+                            ${styles.networkInfo}
+                            ${styles.networkHealthHeading}`}
```

Comment:
What triggered this code styling here? Why not have them inlined ?

Reply:
Do you mean why not put these three lines in a single line? It exceeds the 80 character line length limit enforced by the linter.